### ISSUE TITLE
🐛 Bug Report: Python SDK - Querying with a boolean value leads to " Query type does not match expected: boolean"

### ISSUE THREAD

**Issue by superseby2** [appwrite/sdk-for-python#46]
### 👟 Reproduction steps

Trying to perform a query like so :
```
users_list  = database.list_documents('profiles',                                                                          
                                                                                                                           
[                                                                                                                          
   Query.equal('open', True)                                                                                               
] 
```                                                                                                                         

### 👍 Expected behavior

 - results filtered by "open" being true

### 👎 Actual Behavior

Error "Query type does not match expected: boolean" is raised.

It looks like the Python SDK just pass the value as "True" in the rest api call

https://domain.com/v1/database/collections/profiles/documents?queries%5B0%5D=open.equal%28True%29

converting the value to lowercase will make the query works

Bellow is the part that would probably need some love 
```
appwrite/query.py
 38     def parseValues(value):                                                                                             
 39         if type(value) == str:                                                                                          
 40             return '"{}"'.format(value)                                                                                 
 41         else:                                                                                                           
 43             return value                                                                                               
```
maybe something like ? 
```
appwrite/query.py
 38     def parseValues(value):                                                                                             
 39         if type(value) == str:                                                                                          
 40             return '"{}"'.format(value)                                                                                 
 39         elif type(value) == bool:                                                                                          
 40             return '{}'.format(str(value).lower())                                                                                 
 41         else:                                                                                                           
 43             return value                                                                                               
```

### 🎲 Appwrite version

Version 0.12.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by superseby2** [comment]
@netapy if for some reason you cannot query using the "true" vs "True" trick. What I do is simply not use a boolean field but an int field and query using 0 or 1 ...

**Comment by netapy** [comment]
> 

Thanks for the input. 
I have found the workaround suggested by @stnguyen90 to work very well for my case !

**Comment by eldadfux** [comment]
@abnegate has just merged a fixed, this is expected to be available with the next release of the SDK later this week.

**Comment by superseby2** [comment]
Happy to see this fixed :)

**Comment by abnegate** [comment]
This fix is now released in version `1.0.0` :grin:
